### PR TITLE
[CM-2072] Show answer source url with bot answer 

### PR DESCRIPTION
**Pull request by AbhijeetRanjan308**
## Summary
- Added a new view for Source URL.

## Expected Data In Metadata
```
"sourceURLs" = ["https://dashboard.kommunicate.io", "https://docs.kommunicate.io"]
```

### CODE REVIEW

File: Sources/Controllers/ALKConversationViewController+TableView.swift [hunk 1]
```diff
@@ -74,6 +74,16 @@ extension ALKConversationViewController: UITableViewDelegate, UITableViewDataSou
                 }
                 return cell
 
+            } else if let messageMetadata = message.metadata, let _ = messageMetadata["sourceURLs"], !message.isMyMessage {
```
Comment (Sathyan-KM) [inline Sources/Controllers/ALKConversationViewController+TableView.swift:77]: create a variable for this ` sourceURLs`

File: Sources/ViewModels/ALKConversationViewModel.swift [hunk 2]
```diff
@@ -314,6 +314,12 @@ open class ALKConversationViewModel: NSObject, Localizable {
         switch messageModel.messageType {
         case .text, .html, .email:
             guard !configuration.isLinkPreviewDisabled, messageModel.messageType == .text, ALKLinkPreviewManager.extractURLAndAddInCache(from: messageModel.message, identifier: messageModel.identifier) != nil else {
+                if let messageMetadata = messageModel.metadata, let metadataValue = messageMetadata["sourceURLs"], !messageModel.isMyMessage {
```
Comment (Sathyan-KM) [inline Sources/ViewModels/ALKConversationViewModel.swift:317]: convert raw string into variable

File: Sources/Views/KMFriendSourceURLViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+        } else {
+            avatarImageView.image = placeHolder
+        }
+        sourceLabel.text = localizedString(forKey: "Source",
+                                           withDefaultValue: SystemMessage.LabelName.Source,
+                                           fileName: localizedStringFileName)
+        nameLabel.text = viewModel.displayName
+        var urls = [""]
+        
+        if let messageMetadata = viewModel.metadata, let metadataValue = messageMetadata["sourceURLs"] as? [String] {
```
Comment (Sathyan-KM) [inline Sources/Views/KMFriendSourceURLViewCell.swift:367]: here as well

File: Sources/Controllers/ALKConversationViewController+TableView.swift (hunk 1 above)
Comment (AbhijeetRanjan308) [reply Sources/Controllers/ALKConversationViewController+TableView.swift]: added

File: Sources/ViewModels/ALKConversationViewModel.swift (hunk 2 above)
Comment (AbhijeetRanjan308) [reply Sources/ViewModels/ALKConversationViewModel.swift]: added

File: Sources/Views/KMFriendSourceURLViewCell.swift (hunk 3 above)
Comment (AbhijeetRanjan308) [reply Sources/Views/KMFriendSourceURLViewCell.swift]: added

**Review by Sathyan-KM — APPROVED** [review]

**Pull request merged**